tests/hpc/mpi_master.pm

```diff
@@ -45,7 +45,7 @@ sub run ($self) {
     type_string('pkill -u root', lf => 1) unless $user_virtio_fixed;
     select_user_serial_terminal($prompt);
     # for <15-SP2 the openmpi2 module is named simply openmpi
-    $mpi = 'openmpi' if ($mpi =~ /openmpi2|openmpi3|openmpi4/);
+    my $mpi2load = 'openmpi' if ($mpi =~ /openmpi2|openmpi3|openmpi4/);
```

Comment:
I think variable needs to be declared before this conditional statement to pass CI check.